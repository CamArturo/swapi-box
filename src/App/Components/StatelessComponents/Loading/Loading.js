import React from 'react';
import loading from '../../../assets/darth-vader.mp4';

const Loading = props => {
  return (
    <video src={loading} width="100%" height="100%" autoPlay loop/>
  );
};

export default Loading;
