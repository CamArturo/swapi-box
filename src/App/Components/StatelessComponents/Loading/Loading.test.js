import React from 'react';
import {shallow} from 'enzyme';
import Loading from "./Loading";

it('matchs Snapshot', () => {
  const wrapper = shallow(<Loading/>);

  expect(wrapper).toMatchSnapshot();
});