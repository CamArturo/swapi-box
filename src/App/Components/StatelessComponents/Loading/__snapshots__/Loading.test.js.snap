// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matchs Snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "autoPlay": true,
      "height": "100%",
      "loop": true,
      "src": "darth-vader.mp4",
      "width": "100%",
    },
    "ref": null,
    "rendered": null,
    "type": "video",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "autoPlay": true,
        "height": "100%",
        "loop": true,
        "src": "darth-vader.mp4",
        "width": "100%",
      },
      "ref": null,
      "rendered": null,
      "type": "video",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
